### PR TITLE
ss_* to orb_*

### PR DESCRIPTION
**Pull request by sudughonge**
Addresses https://github.com/poliastro/poliastro/issues/1523

### CODE REVIEW

File: docs/source/examples/Going to Jupiter with Python using Jupyter and poliastro.mystnb [hunk 1]
```diff
@@ -13,7 +13,7 @@ kernelspec:
 
 # Going to Jupiter with Python using Jupyter and poliastro
 
-Let us recreate with real data the [Juno NASSA Mission](https://www.jpl.nasa.gov/news/press_kits/juno/overview/). The main objectives of Juno spacecraft is to study the Jupiter planet: how was its formation, its evolution along time, atmospheric characteristics...
+Let us recreate with real data the [Juno NASSA Mission](https://www.jpl.nasa.gov/news/preorb_kits/juno/overview/). The main objectives of Juno spacecraft is to study the Jupiter planet: how was its formation, its evolution along time, atmospheric characteristics...
```
Comment (astrojuanlu) [inline docs/source/examples/Going to Jupiter with Python using Jupyter and poliastro.mystnb:16]: ⚠️

**Review by astrojuanlu — CHANGES_REQUESTED** [review]
Thanks a lot @sudughonge! 🎉 I only noticed one small mistake, the rest looks good to me!

File: docs/source/examples/Going to Jupiter with Python using Jupyter and poliastro.mystnb (hunk 1 above)
Comment (sudughonge) [reply docs/source/examples/Going to Jupiter with Python using Jupyter and poliastro.mystnb:16]: Whoops, I realized I'd get nonsense replacements. Fixing.

**Comment by astrojuanlu** [comment]
The conflicts are my fault (gh-1540), I'll take care of them

**Comment by astrojuanlu** [comment]
Rebased locally and pushed to a new branch, closing in favor of gh-1542.

**Pull request closed without merging**